tests/integration/DeserializerFactoryTest.php

```diff
@@ -21,9 +23,25 @@ protected function setUp() {
 		$this->factory = TestFactoryBuilder::newDeserializerFactory( $this );
 	}
 
-	public function testGetEntityDeserializerReturnsDeserializer() {
+	public function testNewEntityDeserializerReturnsDeserializer() {
 		$deserializer = $this->factory->newEntityDeserializer();
 		$this->assertInstanceOf( 'Deserializers\Deserializer', $deserializer );
 	}
 
+	public function testNewStatementDeserializerReturnsDeserializer() {
+		$deserializer = $this->factory->newStatementDeserializer();
+		$this->assertInstanceOf( 'Deserializers\Deserializer', $deserializer );
```

Comment:
bumping to PHP to min 5.5 avoids having to use strings for this, as per example at the bottom of https://www.entropywins.wtf/blog/2016/01/30/missing-in-php7-function-references/